packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -522,26 +548,24 @@ export default {
     width: 100%;
 }
 
-@include f.media ('<mid') {
+@include f.media('<mid') {
     .c-cookieBanner-card {
         flex-direction: column;
-        padding: f.spacing(d) 0;
     }
 
     .c-cookieBanner-ios {
         padding-bottom: 80px;
```

Comment:
same here about spacing. I'd also be tempted to swap out `padding-bottom` with `padding-block-end` (might want to double check but I think that's the equivalent)